packages/block-library/src/shortcode/edit.native.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PlainText } from '@wordpress/block-editor';
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
```

Comment:
We can remove this if there are no internal dependencies

Reply:
Removing `Internal dependencies` Causes the Lint error `Expected preceding "Internal dependencies" comment blockeslint(@wordpress/dependency-group)` it doesn't look like `Block code style` is needed though. So I went ahead and removed that instead.